projects/common/vk_renderer.h

```diff
@@ -117,6 +118,14 @@ struct CommandObjects
     ~CommandObjects();
 };
 
+// Descriptor container for convenience
+struct Descriptors
```

Comment:
Should probably just call this `VulkanDescriptorSet` for lack of a better name. `Descriptors` doesn't convey any meaning.